### DISCUSSION TITLE
[Feature Request]: Card update

### DISCUSSION THREAD

**Discussion by duczz** [FaserF/ha-deutschebahn#52] [Q&A]
### The request

Would be great when you can implement this as Card
https://github.com/silviokennecke/ha-custom-components/tree/main/components/public-transport

**Comment by arunoruto** [comment]
That card can already be used with this integration.
Do you have any problems to set it up?

**Comment by duczz** [comment]
Yeah, but it would be great when it is in this Integration directly :)

**Reply by FaserF** [reply]
As arunoruto says, everything here should already be working: 
https://github.com/FaserF/ha-deutschebahn?tab=readme-ov-file#lovelace-custom-card

For any issues with the Lovelace card you can create an issue on the corresponding repository or use the discussions feature to get support from the community, as this has nothing directly to do with this integration here.

**Comment by duczz** [comment]
the card works yes, but only when i install the other repo, what is when the other repo goes off.
i think it would be better and safer when it will be integrate in this repo directly.

or im blind and dont see it in this integratin?

**Reply by FaserF** [reply]
No you are correctly writing how it works. But this here is an integration. The card is a Lovelace card. Those are two separate things, that should be kept separately. 

Also I am not using the Lovelace card by myself and therefore I have no interest in maintaining it :) as I can't reproduce Lovelace issues. Lovelace cards are also written in a complete different programming language (JavaScript vs Python)

**Reply by duczz** [reply]
Ah ok, than its fine for me :) - thanks
